docs/src/add-services/influxdb.md

```diff
@@ -28,41 +28,36 @@ The format exposed in the ``$PLATFORM_RELATIONSHIPS`` [environment variable](../
 
 {{% endpoint-description type="influxdb" /%}}
 
-```php
-<?php
-// This assumes a fictional application with an array named $settings.
-if (getenv('PLATFORM_RELATIONSHIPS')) {
-	$relationships = json_decode(base64_decode($relationships), TRUE);
-
-	// For a relationship named 'influxtimedb' referring to one endpoint.
-	if (!empty($relationships['influxtimedb'])) {
-		foreach ($relationships['influxtimedb'] as $endpoint) {
-			$settings['influxdb_host'] = $endpoint['host'];
-			$settings['influxdb_port'] = $endpoint['port'];
-			break;
-		}
-	}
-}
-```
+{{< codetabs >}}
+
+---
+title=PHP
+file=static/files/fetch/examples/php/influxdb
+highlight=php
+---
+
+{{< /codetabs >}}
 
 ## Exporting data
 
-InfluxDB includes its own [export mechanism](https://docs.influxdata.com/influxdb/v1.2/tools/influx_inspect/).
-To gain access to the server from your local machine open an SSH tunnel with the Platform.sh CLI:
+To export your data from InfluxDB:
 
-```bash
-platform tunnel:open
-```
+1. Install the [influx-cli](https://docs.influxdata.com/influxdb/cloud/tools/influx-cli/).
+2. Gain access to the server from your local machine with the [Platform.sh CLI](../administration/cli/_index.md):
 
-That opens an SSH tunnel to all services on your current environment and produce output like the following:
+	```bash
+	platform tunnel:open
+	```
 
-```bash
-SSH tunnel opened on port 30000 to relationship: influxtimedb
-```
+	That command opens an SSH tunnel to all services on your current environment and produces output like:
 
-The port may vary in your case.
-Then run InfluxDB's export commands as desired.
+	```bash
+	SSH tunnel opened on port {{<variable "PORT" >}} to relationship: influxtimedb
+	```
+3. Get the username and password [for your service](../development/variables/use-variables.md#access-variables-in-a-shell) with `echo $PLATFORM_RELATIONSHIPS | base64 --decode`.
```

Comment:
:lady_beetle: The link text here is rather confusing compared to what the link actually goes to.

Reply:
:hammer_and_wrench: 
```suggestion
3. Get the username and password for your service from an [environment variable](../development/variables/use-variables.md#access-variables-in-a-shell)
by running `echo $PLATFORM_RELATIONSHIPS | base64 --decode`.
```

Reply:
:hammer_and_wrench This command doesn't work without other steps in the middle (such as running `export PLATFORM_RELATIONSHIPS="$(platform tunnel:info --encode)"`). It seems much easier to use the CLI, since that's already a prerequisite.
````suggestion
3.  Get the username and password from the [relationship](#relationship) by running the following command:

    ```bash
    platform relationships -P {{<variable "RELATIONSHIP_NAME" >}}
    ```

````